chips/src/main/java/com/decathlon/vitamin/compose/chips/VitaminChipRipples.kt

```diff
@@ -0,0 +1,26 @@
+package com.decathlon.vitamin.compose.chips
+
+import android.os.Build
+import androidx.compose.material.ripple.RippleAlpha
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+// todo to delete
+internal object VitaminChipRipples {
```

Comment:
Ripple effect has been migrated in foundation module. You can't reused one of them? Or create a new one there?

Reply:
I changed for VitaminTheme Ripples